### ISSUE TITLE
Throw error if fetching from relative URL on the server

### ISSUE THREAD

**Issue by shuding** [vercel/satori#203]
# Feature Request

## Description

If running on the server, you cannot use `<img src="/my-img.png"/>` because there is no domain provided. We should throw a specific error message in that case.

**Comment by wong2** [comment]
`node-fetch` already throw such an error: `Can't load image /aaa.jpg: Only absolute URLs are supported`

Node.js 18: `Can't load image /aaa.jpg: Failed to parse URL from /aaa.jpg`

**Comment by shuding** [comment]
We need to unify that error message, also I think some fetch polyfills won't throw in that case.

**Comment by KiwiKilian** [comment]
Not quite exactly on the topic, but is there any other option to fix this error other than inlining the image?

**Comment by shuding** [comment]
You can fetch the image with an absolute URL such as `<img src="https://example.com/my-img.png"/>`.

**Comment by KiwiKilian** [comment]
Thanks for the answer, I should have been more clear. I was thinking about other solutions to local usage on the server. For example a `resolveResource` function one could pass as an option?

**Comment by multipletwigs** [comment]
Hey! I'd love to take this issue (for my first OSS contribution) if it hasn't been assigned yet 🥳

**Comment by multipletwigs** [comment]
Hey there, spent some time tinkering around the codebase, and I was wondering if the enhancement is as simple as checking the `src` with the following code. First checking if the code is running on a server environment, then checking if it's using an absolute url. 

```ts
// In src/handler/image.ts

// Check if on server
  if (typeof window === 'undefined') {
    // Check if src is absolute url
    if (!src.startsWith('http') && !src.startsWith('data:')) {
      throw new Error(`Image source must be an absolute URL: ${src}`)
    }
  }
```

**Comment by shuding** [comment]
@multipletwigs that fix makes sense to me! Feel free to open a PR :)

**Comment by shuding** [comment]
Fixed with #447.
